client/editSchedule.js

```diff
@@ -261,6 +313,27 @@ Template.editSchedule.rendered = function() {
   setupCompetitionCalendar(template, $calendar, $addEditSomethingModal);
 };
 
+Template.unscheduledRound.rendered = function() {
+  var template = this;
+  var $unscheduledRound = template.$('.fc-event');
+
+  // store data so the calendar knows to render an event upon drop
+  // $unscheduledRound.data('event');
+
+  // make the event draggable using jQuery UI
+  $unscheduledRound.draggable({
+    zIndex: 999,
+    revert: false,      // will cause the event to go back to its
```

Comment:
revert false means the event will go back to its original position? Is this comment stale?